appnav/src/test/kotlin/io/element/android/appnav/loggedin/LoggedInPresenterTest.kt

```diff
@@ -73,6 +75,8 @@ class LoggedInPresenterTest {
             networkMonitor = FakeNetworkMonitor(networkStatus),
             pushService = FakePushService(),
             sessionVerificationService = FakeSessionVerificationService(),
+            analyticsService = FakeAnalyticsService(),
+            encryptionService = FakeEncryptionService()
```

Comment:
Could we add some test code to check these 2 are being used?

Reply:
Added